runtime/onert/core/include/ir/operation/Bulk.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_OPERATION_BULK_H__
+#define __ONERT_IR_OPERATION_BULK_H__
+
+#include "ir/Operation.h"
+
+namespace onert
+{
+namespace ir
+{
+namespace operation
+{
+
+class Bulk : public Operation
+{
+
+  struct Param
+  {
+    std::string binary_path;
+    std::string info_path;
```

Comment:
Does It imply that 

1) information about prebuilt will come from nnpkg `configs` ?
2) validation will not be done in loader, but in backend ?

Reply:
(Q) What does the `info_path` mean?

Reply:
info_path is reserved for future use for now. It is not mandatory.

Reply:
@chunseoklee

About my Q1: I understood. What about removing this at this moment and adding when it become necessary.

About my Q2: Not answered yet. I guess frontend does validation, backend will get every information by itself from `binary_path`. Is it right?

Reply:
> What about removing this at this moment and adding later.

OK.  i will update PR

> validation will not be done in loader, but in backend ?

I prefer loader. 

> From this PR, backend will get every information by itself from binary_path. Is it right?
For details, I failed to figure out from this PR. If it is not clear, it would be better not to merge this PR yet.

I do not get the point. Could you please describe more ?